src/string/README.md

```diff
@@ -10,6 +10,14 @@ __Properties__
 
 [kmp-wiki]: https://en.wikipedia.org/wiki/Knuth–Morris–Pratt_algorithm
 
+### [Manacher](./manacher.rs)
+From [Wikipedia][manacher-wiki]: find a longest palindrome in a string in linear time.
+
+__Properties__
+* Case performance  O(n)
+* Case space complexity  O(n)
```

Comment:
```suggestion
* Worst-case time complexity is O(n)
* Worst-case space complexity is O(n)
```

Reply:
@siriak 
If you suggest changing so, I recommend you changing below as well.

You should keep same format.

https://github.com/TheAlgorithms/Rust/blob/master/src/string/README.md

Reply:
Agree, let's change it everywhere